### PR TITLE
Attempt to fix potential race condition with em-http-request

### PR DESCRIPTION
**Pull request by jkodumal**

### CODE REVIEW

File: lib/ldclient-rb/stream.rb [hunk 1]
```diff
@@ -136,7 +136,7 @@ def boot_event_manager
         set_disconnected
       end
       source.inactivity_timeout = 0
-      source.start
+      EM.schedule{ source.start }
```
Comment (jhubert) [inline lib/ldclient-rb/stream.rb]: Stylistically it's good to have a space before the `{`

```ruby
EM.schedule { source.start }
```

**Comment by jkodumal** [comment]
I've also got to fix the tests on this one-- just getting the PR up so that @jhubert can test this out.

**Comment by jhubert** [comment]
Installed locally. About to head for a lunch meeting but I'll see if we end up in that state when i get back.

**Comment by jkodumal** [comment]
This did not address the issue-- in addition we're moving forward with Celluloid instead of EventMachine.

**Pull request closed without merging**